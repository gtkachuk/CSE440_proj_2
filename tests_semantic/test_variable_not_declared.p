program testVariableNotDeclared;

class testVariableNotDeclared
BEGIN

   VAR 
      compilerWorks : boolean;
      
FUNCTION testVariableNotDeclared;
BEGIN
   jobDone := 1
END

END
.

