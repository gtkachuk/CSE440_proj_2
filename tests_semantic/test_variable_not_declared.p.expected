ERROR:12:Variable "jobDone" not declared
Errors detected. Exiting.
